examples/iris/iris.ipynb

```diff
@@ -1,323 +1,503 @@
 {
- "cells": [
-  {
-   "cell_type": "code",
-   "execution_count": 1,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "from foundry import Foundry"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "# Instantiate Foundry from local file and load data"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 2,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "#need Globus to pull data\n",
-    "f = Foundry().load(\"_test_blaiszik_foundry_iris_v2.1\")\n",
-    "X,y = f.load_data()"
-   ]
+  "nbformat": 4,
```

Comment:
At the end, I think the title *Only Run to Create Local Metadata and Dataframe* should be changed to *Create Local Metadata and Dataframe* and then a sentence should be added below that says something like: 
Shape your data (in this case, the Iris dataset from Scikit-Learn) into a dataframe and save it as JSON if you want to publish it to Foundry later (see other notebooks on how to publish datasets). Save the file to 'foundry_dataframe.json'

Similarly, you need to create a metadata file for your dataset to tell Foundry what your data is all about! Format your metadata like so, and then save it to 'foundry_metadata.json'